projects/pt1/e2e_testing/xfail_sets.py

```diff
@@ -498,7 +503,12 @@
     "ViewCollapseDynamicWithAtenSizeIntModule_basic",
     "ViewSizeFromOtherTensor_basic",
     "WeightNormInterfaceModule_basic",
+    # REMOVE ON WHEN ENABLE_GQA IS ADDED
```

Comment:
Isn't enable_gqa already added?

Reply:
Not on stable (v2.4.0), it works on v2.5.0

Reply:
Can we not just change the test to use `attn_mask=mask` then we don't need to care about GQA for now.

Reply:
It's expecting GQA while torch doesn't have it yet, we need to wait for that